demo/index.html

```diff
@@ -126,9 +129,12 @@
       output.innerHTML = JSON.stringify(event.detail);
     }
 
-    function clickHandler(event) {
+    function submitHandler(event) {
       Polymer.dom(event).localTarget.parentElement.submit();
     }
+    function resetHandler(event) {
+      Polymer.dom(event).localTarget.parentElement.reset();
```

Comment:
I think you technically still need to wrap the `localTarget` in a call to `Polymer.dom(...)` for this to be correct for all DOM arrangements where ShadeyDOM is a factor.